modules/services/window-managers/i3-sway/sway.nix

```
@@ -556,6 +561,18 @@ in
         '';
       };
 
+      dbusImplementation = mkOption {
+        type = types.enum [
+          "dbus"
+          "broker"
+        ];
+        default = "dbus";
+        example = "broker";
+        description = ''
+          The implementation to use for the D-Bus activation.
```

I think this could use more explanation as to what happens for each of the values.
My suggestion:

```suggestion
          The D-Bus implementation used on the system. This affects which tool is used to import environment variables during startup.
          On NixOS, this should match the value of the option [`services.dbus.implementation` (NixOS)](                            https://nixos.org/manual/nixos/stable/options#opt-services.dbus.implementation)
          When set to `dbus`, `dbus-update-activation-environment --systemd <variables>` is run.
          Otherwise, when set to`broker`, `systemctl --user import-environment <variables>` is run.

          See <https://github.com/swaywm/sway/wiki#systemd-and-dbus-activation-environments> for more documentation.
```